build/BuildDefaults.props

```diff
@@ -7,5 +7,14 @@
     <UsePortableLinuxSharedFramework Condition=" '$(UsePortableLinuxSharedFramework)' == '' AND '$(OSPlatform)' == 'linux' ">true</UsePortableLinuxSharedFramework>
     <IncludeSharedFrameworksForBackwardsCompatibilityTests Condition=" $(IncludeSharedFrameworksForBackwardsCompatibilityTests) == '' AND '$(Rid)' != 'linux-x64' ">true</IncludeSharedFrameworksForBackwardsCompatibilityTests>
     <HighEntropyVA>true</HighEntropyVA>
+
+    <!-- Only use asset target fallback that we set (not implicit one to net461). -->
+    <DisableImplicitAssetTargetFallback>true</DisableImplicitAssetTargetFallback>
+
+    <!-- Disable asset target fallback warning globally since it does not work transitively on NoWarn of individual packages -->
```

Comment:
That sounds like a bug

Reply:
It is. https://github.com/NuGet/Home/issues/5501

IMHO, nowarn globally on this is OK of you only use AssetTargetFallback explicitly. That's why I'm not flagging a to-do on this. We've got bigger fish to fry.